src/irmin-pack/config.ml

```diff
@@ -51,14 +51,33 @@ let throttle_converter : throttle Irmin.Private.Conf.converter =
   in
   (parse, print)
 
+type freeze_throttle = [ index_throttle | `Cancel_existing ]
```

Comment:
the configuration arguments for irmin_pack_layers are defined here https://github.com/mirage/irmin/blob/master/src/irmin-pack/irmin_pack_layers.ml#L55; but maybe we should move them all in `config.ml`?